.github/workflows/test_adapters.yaml

```diff
@@ -25,9 +25,6 @@ on:
       patternfile_name:
         required: true
         type: string
-      k8s_version:
-        required: true
-        type: string
```

Comment:
This will remove backwards compatibility. We will need another sweeping change across the adapters. I think it's better to keep using string parsed arrays like the other fields. What do you think @4molybdenum2?

Reply:
Hmm, thats true...
Currently we cant pass array types as workflow inputs... so another workaround maybe passing strings and parsing them according to some delimiter (eg: comma).

On the other hand if we pass the k8s version using matrix then the same k8s version will be used to test against all the adapters. As for the backwards compatitbility I don't think the tests will fail because the k8s_version taken from the inputs are all replaced by ```matrix.k8s_version```. So, it will not technically break the CI tests if I am correct..? But yes, we will have to remove all those k8s_version fields from the individual adapters.

Reply:
It does fail, I'm afraid. Have a look at https://github.com/gr455/meshery-istio/actions/runs/2401529126
It expects the input field to be in the core workflow

Reply:
Ah, I see...then let me see how to do it by passing a string of versions. I guess that's one way to tackle it

Reply:
Sounds good!

Reply:
@gr455 I just made the k8s_version a field and set it to ```not required```. Can you check if it works now?